### DISCUSSION TITLE
PHPMailer sent but email not received

### DISCUSSION THREAD

**Discussion by farhan-hafizh** [PHPMailer/PHPMailer#2622] [Q&A] [answered]
So i write this simple function
```
function sendMail($email,$pdf){
    $mail = new PHPMailer;

$mail->From = "no-reply@test.com";
$mail->FromName = "Client";
$mail->addAddress($email, "User");     // Add a recipient
$mail->addReplyTo("marketing@test.com", "Support");


$mail->addStringAttachment($pdf, "test.pdf");

$mail->AddEmbeddedImage("logo.png", "logo");

$mail->isHTML(true);                               

$mail->Subject = "This Is A Test";
$mail->Body    = "Hello this is a test";

if(!$mail->send()) {
    echo "Message could not be sent.";
    echo "Mailer Error: " . $mail->ErrorInfo;
} else {
    echo "Message has been sent";
}
}
```
The result:
![image](https://user-images.githubusercontent.com/64812051/152116147-274fd678-6d11-4dfa-a046-221e27521741.png)

But it didn't sent anything or my test email didn't get any email. Anyone can give me an idea why and how to solve it?

**Comment by Synchro** [comment, marked as answer]
You're not calling `isSMTP()`, which means PHPMailer will send using PHP's built-in `mail()` function, which submits messages via a shell to a local mail server. After submitting a message successfully, the mail server is then responsible for onward delivery, and PHPMailer plays no part in that. So, to find out what is happening to your messages, check your local mail server's logs, usually somewhere like `/var/log/mail.log`; any undelivered messages will probably be somewhere in `/var/spool/mail`.

**Reply by farhan-hafizh** [reply]
But i don't want to use SMTP? Is that really necessary to add `isSMTP()` even I don't want to use it?

**Reply by Synchro** [reply]
I didn't say that. It happens that it is both faster and safer to use SMTP to localhost than to use `mail()`, but that's a separate point. What I said was that PHPMailer is working just fine, but your mail server is not, so fix your mail server.

**Reply by farhan-hafizh** [reply]
What happen though? It was okay like a month ago and i didn't change anything but now it's not. I'm really but i have no clue at all for now T_T

**Reply by Synchro** [reply]
You will find out if you look at your mail server logs...
